.drone.star

```diff
@@ -1456,7 +1456,7 @@ def acceptance(ctx):
                             steps += buildGithubCommentForBuildStopped(suiteName)
 
                         # Upload the screenshots to github comment
-                        steps += githubComment(alternateSuiteName)
+                        steps += githubComment(alternateSuiteName, server)
```

Comment:
I would suggest this so that we will have these comments: `e2e-tests oC10`, `e2e-tests oCIS`, `acceptance oC10` and `acceptance oCIS`

```suggestion
                        server_type = "oCIS" if params["runningOnOCIS"] else "oC10"
                        steps += githubComment("acceptance", server_type)
```